GiniVision/Classes/CameraViewController.swift

```diff
@@ -17,13 +17,28 @@ import AVFoundation
 public typealias CameraSuccessBlock = (_ imageData: Data) -> ()
 
 /**
- Block which will be executed if an error occurs on the camera screen. It contains a camera specific error.
+ Block which will be executed when the camera screen successfully takes a picture or pick a document/picture. It contains the JPEG representation of the image including meta information about the image, or the PDF Data. Should be indicated if the document was imported or not.
```

Comment:
`Should be indicated if the document was imported or not`
That's not good wording... sounds funny. But I hope it won't be needed - see my next comment.